### ISSUE TITLE
No gem published

### ISSUE THREAD

**Issue by glenux** [CodeOne45/wordle-tui#1]
It would be awesome to have the app installable with a simple

    gem install wordle-tui

And playable with 

    wordle-tui

**Comment by CodeOne45** [comment]
> It would be awesome to have the app installable with a simple
> 
> ```
> gem install wordle-tui
> ```
> 
> And playable with
> 
> ```
> wordle-tui
> ```

Yes I thought of that but I had a hard time understanding how it works.

**Comment by glenux** [comment]
To create a gem, you first need to create a `wordle-tui.gemspec` file.

The easiest way to create such a file is to type `bundle gem wordle-tui` and copy/paste generated files into your project.
